addons/website/static/src/snippets/s_product_list/000.scss

```diff
@@ -28,19 +28,19 @@
             > .btn {
                 width: 100%;
                 padding: 5px !important;
-                font-size: 16px;
+                @include font-size(16px);
 
                 @media only screen and (max-width : 1280px) { // FIXME
-                    font-size: 12px;
+                    @include font-size(12px);
```

Comment:
Same: inside a media query, does this even work? But even if it does, it's useless no? Please check all your PR again :wink: